src/gentropy/susie_finemapper.py

```diff
@@ -82,9 +121,9 @@ def susie_finemapper_one_studylocus_row(
             .filter(f.col("z").isNotNull())
         )
         # Remove ALL duplicated variants from GWAS DataFrame - we don't know which is correct
-        variant_counts = gwas_df.groupBy("variantId").count()
-        unique_variants = variant_counts.filter(f.col("count") == 1)
-        gwas_df = gwas_df.join(unique_variants, on="variantId", how="left_semi")
+        # variant_counts = gwas_df.groupBy("variantId").count()
```

Comment:
Why did you comment these lines?

Reply:
Was doing some testing to see how long it added to computation time - uncommented now.